docs/pages/more/expo-cli.mdx

```diff
@@ -326,6 +326,7 @@ For example, you are testing a new version of `react-native-reanimated` to verif
 You can exclude specific packages from the version checks while still allowing the `install` command to install, check, and fix any other dependencies. This configuration extends to the checking done by `npx expo-doctor`.
 
 To exclude packages from version checking, set the `expo.install` config object in your project's **package.json**:
+
```

Comment:
I ran "format" on this page, which added this, but I can remove it 🤷 (but it seems like a good addition, separating code blocks with text a bit nicer)